sw/device/boot_rom/BUILD

```diff
@@ -0,0 +1,78 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules:opentitan.bzl", "OPENTITAN_CPU", "opentitan_binary")
+load("//rules:autogen.bzl", "autogen_chip_info")
+
+autogen_chip_info(
+    name = "chip_info",
+)
+
+opentitan_binary(
+    name = "boot_rom",
+    linkopts = [
+        "-nostartfiles",
+        "-nostdlib",
+    ],
+    output_scrambled = True,
+    deps = [
+        ":boot_rom_lib",
```

Comment:
I think we need `base:mmio` here to force linking with a concrete mmio implementation.

Reply:
Yes, but why here and not in boot_rom_lib? It's the first thing along the demendency chain to depend on mmio.h and to be OPENTITAN_CPU specific AFAIK.

Also, I'm trying to figure out why I didn't see a linker error without that. I was hoping to see those sorts of things pop up during link time.

Reply:
It should _also_ work in `boot_rom_lib` and it should really make too much of a difference.

I think I leaned towards `boot_rom` because its the last opportunity to bind the specific implementation you want.